iocextractor/core/server.py

```diff
@@ -4,7 +4,6 @@
 
 # pylint: disable=C0413, C0411
 
-from logging import debug
```

Comment:
Remove this line

Reply:
Do you mean I should undo the removing of the debug import?

Reply:
Yes remove the import statement

Reply:
I thought I already did by the first commit.

Reply:
Yes fine.